packages/flutter/test/widgets/form_test.dart

```diff
@@ -1126,4 +1126,61 @@ void main() {
     await tester.pump();
     expect(find.text('test_error'), findsNothing);
   });
+
+  testWidgets('AutovalidateMode.onUnfocus', (WidgetTester tester) async {
```

Comment:
Can you add another test for the case where the Form and FormField have conflicting AutovalidateModes?